### DISCUSSION TITLE
Always skip Almond automation dialog

### DISCUSSION THREAD

**Discussion by noxhirsch** [home-assistant/frontend#7642] [Automations / scripts / scenes] [answered]
Everytime a user wants to create an automation with the UI editor, the Almond dialog appears first - this can be pretty annoying if you don't use it (especially if you can't use it because you don't speak English). It would be nice to have an option to "always skip" this dialog or to completely disable this function.

![image](https://user-images.githubusercontent.com/30938717/98728832-4b49de00-239a-11eb-944b-04b666a8f1fd.png)

**Comment by spacegaier** [comment, marked as answer]
This discussion will be void with the new HA version 0.119 that will include blueprints, since that version provides a new dialog for automation creation tied to it.

**Reply by Mincka** [reply]
I just hope it will not replace this dialog with another one to choose a blueprint that cannot be always skipped.

**Reply by frenck** [reply]
From 1.0.0b0 it looks like this:
![image](https://user-images.githubusercontent.com/195327/101278408-5803fb00-37bb-11eb-8154-86f71e7bea47.png)

**Reply by Mincka** [reply]
I understand the need to highlight blueprints but I am not sure that's something I want to be proposed each time I add an automation.

Now there are labels on "+" buttons, I think it could have been useful to have two "+" buttons:
* \+ Add Automation
* \+ Add Blueprint Automation

The flow would be more natural since you could go directly to the screen you want in one click.

Anyway, even if the extra click is still there, it _feels_ less intrusive. The button at the right to "ignore" is less cumbersome than the previous "Skip" on the left.
